### PR TITLE
✍🏻 Fix docs

### PR DESCRIPTION
**Pull request by JuaniRios**
## What?
- Initial fixing of stale documentation on pallet-funding

### CODE REVIEW

**Comment by JuaniRios** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/386?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#387** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/387?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#386** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/386?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#383** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/383?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#384** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/384?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#385** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/385?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JuaniRios and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by lrazovic** [comment]
@simonpolimec @KenPolimec Can you help us with the review of this tech doc?

File: pallets/funding/src/lib.rs [hunk 1]
```diff
@@ -18,96 +18,51 @@
 //!
 //! Polimec's main business logic. It allows users to create, evaluate, and fund projects.
 //!
-//! It rewards project evaluators and contributors with `Contribution Tokens`. These tokens
-//! can be redeemed for a project's native tokens, after their parachain is deployed on mainnet.
-//! ## 👷 Work in progress 🏗️
-//! Expect major changes between PRs
+//! It rewards project participants with Contribution Tokens. These tokens
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:21]: Participants get contribution tokens. Contribution tokens are tokens issued by projects which successfully raised funds on Polimec. They are distributed to evaluators and participants who contributed to the project’s successful funding round. Contribution tokens are transferability-locked in the wallet of participants or evaluators after distribution and are automatically converted to the project’s transferable mainnet token at launch.

File: pallets/funding/src/lib.rs [hunk 2]
```diff
@@ -18,96 +18,51 @@
...
+//! can be redeemed for a project's native tokens once the project goes live.
 //!
 //! ## Overview
-//! The official logic for Polimec's blockchain can be found at our [whitepaper](https://polimec.link/whitepaper).
+//! The official logic for Polimec's blockchain can be found at our [knowledge hub](https://hub.polimec.org/).
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:25]: [...] can be found in [...] (replace "at" with "in")

File: pallets/funding/src/lib.rs [hunk 3]
```diff
@@ -18,96 +18,51 @@
...
 //!
 //! There are 3 types of users in Polimec:
 //! - **Issuers**: They create projects and are responsible for their success.
-//! - **Evaluators**: They evaluate projects and are rewarded for their work.
-//! - **Contributors**: They contribute financially to projects and are rewarded on the basis of their contribution
+//! - **Evaluators**: They evaluate projects by locking PLMC and give them access to funding rounds. They are rewarded if the project gets funded, and slashed otherwise.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:29]: By locking PLMC, evaluators on Polimec are incentivized to assess projects accurately, as they earn rewards for successful evaluations or get slashed for unsuccessful ones.

File: pallets/funding/src/lib.rs [hunk 4]
```diff
@@ -18,96 +18,51 @@
...
+//! - **Contributors**: They contribute financially to projects by locking PLMC and paying out USDT/USDC/DOT, and are rewarded in Contribution Tokens.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:30]: There is no such thing as "contributors", I guess you mean "participants"?

Participants on Polimec are categorized as retail, professional, or institutional investors. They contribute to project funding rounds and receive contribution tokens in return, which are eventually converted into mainnet tokens when the project goes live.

File: pallets/funding/src/lib.rs [hunk 5]
```diff
@@ -18,96 +18,51 @@
...
 //!
-//! A contributor, depending on their investor profile, can participate in different rounds of a project's funding.
+//! A user, depending on their investor profile, can participate in different ways
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:32]: I don't know what you want to explain with "user"? Is it maybe part of "Participants"? If yes, then I would delete it.

File: pallets/funding/src/lib.rs [hunk 6]
```diff
@@ -18,96 +18,51 @@
...
 //!
-//! There are 3 types of contributors:
-//! - **Institutional**
-//! - **Professional**
-//! - **Retail**
+//! There are 3 types of investors, which are authenticated by a KYC/AML process and a credential minted on the KILT blockchain:
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:34]: There are 3 types of participants. A trusted third party attests and classifies them into the following categories based on their financial circumstances, governance status, or professional experience:

File: pallets/funding/src/lib.rs [hunk 7]
```diff
@@ -18,96 +18,51 @@
...
+//! - **Institutional**: has access to creating projects, bidding, and contributing.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:35]: has access to project creation, bidding, and contribution 

(please be aware to remove the dot)

File: pallets/funding/src/lib.rs [hunk 8]
```diff
@@ -18,96 +18,51 @@
...
-//! | `vested_plmc_bid_unbond_for`                  |        |                 | X                     | X                      |
-//! | `vested_plmc_purchase_unbond_for`             |        | X               | X                     | X                      |
-//! | `vested_contribution_token_bid_mint_for`      |        |                 | X                     | X                      |
-//! | `vested_contribution_token_purchase_mint_for` |        | X               | X                     | X                      |
-//!
-//! _* They can call contribute only if the project is on the remainder round._
-//!
+//! 1) **Project Creation**: Issuer creates a project with the [`create_project`](Pallet::create_project) extrinsic.
+//! 2) **Evaluation Start**: Issuer starts the evaluation round with the [`start_evaluation`](Pallet::start_evaluation) extrinsic.
+//! 3) **Evaluate**: Evaluators bond PLMC on the project with the [`bond_evaluation`](Pallet::evaluate) extrinsic.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:42]: Evaluators bond PLMC to evaluate a project with the [`bond_evaluation`](Pallet::evaluate) extrinsic.

File: pallets/funding/src/lib.rs [hunk 9]
```diff
@@ -18,96 +18,51 @@
...
+//! 4) **Evaluation End**: Anyone can end the evaluation round with the [`end_evaluation`](Pallet::end_evaluation) extrinsic after the defined end block.
+//! 5) **Auction Start**: If the project got >=10% of its requested funding (USD) in PLMC bonded, the auction starts immediately after `evaluation_end` was called.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:44]: **Auction Start**: If the project receives at least 10% of its target funding (in USD) in PLMC bonded, the auction starts immediately after `evaluation_end` is called.

File: pallets/funding/src/lib.rs [hunk 10]
```diff
@@ -18,96 +18,51 @@
...
+//! 6) **Bid**: Professional and Institutional investors can bid on the project with the [`bid`](Pallet::bid) extrinsic. Price starts at the issuer defined minimum price, and goes up in a price delta and bucket size of 10%
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:45]: **Bid**: Professional and institutional participants can place bids on the project using the [`bid`](Pallet::bid) extrinsic. The price starts at the issuer-defined minimum and increases by increments of 10% in price and bucket size.

File: pallets/funding/src/lib.rs [hunk 11]
```diff
@@ -18,96 +18,51 @@
...
+//! 7) **Auction End**: Anyone can end the auction round with the [`end_auction`](Pallet::end_auction) extrinsic after the defined end block.
+//! 8) **Community Round Start**: After `end_auction` is called, a weighted average price is calculated from the bids, and the community round starts.
+//! 9) **Contribute**: Anyone without a winning bid can now buy tokens at the calculated price with the [`contribute`](Pallet::contribute) extrinsic.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:48]: **Contribute**: Anyone without a winning bid can now contribute at the weighted average price with the [`contribute`](Pallet::contribute) extrinsic.

File: pallets/funding/src/lib.rs [hunk 12]
```diff
@@ -18,96 +18,51 @@
...
+//! 10) **Remainder Round Start**: After a defined [period](<T as Config>::CommunityRoundDuration) , the remainder round starts.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:49]: **Remainder Round Start**: After a defined [period](<T as Config>::CommunityRoundDuration), even participants with winning bids can contribute.

File: pallets/funding/src/lib.rs [hunk 13]
```diff
@@ -18,96 +18,51 @@
...
+//! 11) **Contribute**: Participants with winning bids can also now buy tokens at the calculated price with the [`contribute`](Pallet::contribute) extrinsic.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:50]: **Contribute**: Participants with winning bids can also contribute at the weighted average price with the [`contribute`](Pallet::contribute) extrinsic.

File: pallets/funding/src/lib.rs [hunk 14]
```diff
@@ -18,96 +18,51 @@
...
+//! 12) **Funding End**: Anyone can end the project with the [`end_project`](Pallet::end_project) extrinsic after the defined end block.
+//! The project will now be considered Failed if it reached <33% of its target funding in USD, and Successful otherwise.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:52]: small change we made, it is now: ≤33%

https://app.gitbook.com/o/DylIO6OKO0VzgNC2IHRI/s/u7OcOgTxjCC949cKG4Hu/learn/funding-process/evaluation

File: pallets/funding/src/lib.rs [hunk 15]
```diff
@@ -18,96 +18,51 @@
...
+//! 13) **Settlement Start**: Anyone can start the settlement process with the [`start_settlement`](Pallet::start_settlement) extrinsic after the defined end block.
+//! 14) **Settle Evaluation**: Anyone can now settle an evaluation with the [`settle_evaluation`](Pallet::settle_evaluation) extrinsic.
+//! This will unlock the PLMC bonded, and either apply a slash or reward come CTs to the evaluator.
+//! 15) **Settle Bid**: Anyone can now settle a bid with the [`settle_bid`](Pallet::settle_bid) extrinsic.
+//! This will set a vesting schedule on the PLMC bonded, and pay out the funding assets to the issuer. It will also issue refunds in case the bid failed,
+//! or the price paid was higher than the final price.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:58]: here, you mean "was higher than the weighted average price", right?

File: pallets/funding/src/lib.rs [hunk 16]
```diff
@@ -18,96 +18,51 @@
...
+//! 16) **Settle Contribution**: Anyone can now settle a contribution with the [`settle_contribution`](Pallet::settle_contribution) extrinsic.
+//! This will set a vesting schedule on the PLMC bonded, and pay out the funding assets to the issuer.
+//! 17) **Settlement End**: Anyone can now mark the project settlement as finished by calling the [`mark_project_as_settled`](Pallet::mark_project_as_settled) extrinsic.
+//! 18) **Migration Start**: Once the issuer has tokens to distribute on mainnet, he can start the migration process with the [`start_offchain`](Pallet::start_offchain_migration) extrinsic.
+//! 19) **Confirm Migration**: For each participant, the issuer must mark their CTs as migrated with the [`confirm_offchain_migration`](Pallet::confirm_offchain_migration) extrinsic.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:63]: **Confirm Migration**: The issuer has to mark each participant's CTs as migrated with the [`confirm_offchain_migration`](Pallet::confirm_offchain_migration) extrinsic.

**Review by simonpolimec — COMMENTED** [review]
reviewed by SIRI

File: pallets/funding/src/lib.rs [hunk 17]
```diff
@@ -18,96 +18,53 @@
...
+//! The official logic for Polimec's blockchain can be found in our [knowledge hub](https://hub.polimec.org/).
 //!
 //! There are 3 types of users in Polimec:
 //! - **Issuers**: They create projects and are responsible for their success.
-//! - **Evaluators**: They evaluate projects and are rewarded for their work.
-//! - **Contributors**: They contribute financially to projects and are rewarded on the basis of their contribution
+//! - **Evaluators**: They are incentivized to assess projects accurately by locking their PLMC. If enough PLMC is locked,
+//! a project is given access to the funding round. Evaluators are either rewarded in contribution tokens if the project gets funded,
+//! or have their PLMC slashed otherwise.
+//! - **Participants**: They contribute financially to projects by locking PLMC and paying out USDT/USDC/DOT, and are rewarded in Contribution Tokens.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:34]: for consistency reasons, would write "contribution tokens" (small letters, not capital ones)

File: pallets/funding/src/lib.rs [hunk 18]
```diff
@@ -18,96 +18,53 @@
...
 //!
-//! A contributor, depending on their investor profile, can participate in different rounds of a project's funding.
-//!
-//! There are 3 types of contributors:
-//! - **Institutional**
-//! - **Professional**
-//! - **Retail**
+//! Users need to go through a KYC/AML by a third party in order to use the protocol. This process classifies them
+//! into one of the following categories, based on their investment experience and financial status:
+//! - **Institutional**: Can take the role of issuer, evaluator, and participant
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:38]: for consistency reasons, add a dot at the end (such as with the other rows below

File: pallets/funding/src/lib.rs [hunk 19]
```diff
@@ -18,96 +18,53 @@
...
-//! | `vested_contribution_token_bid_mint_for`      |        |                 | X                     | X                      |
-//! | `vested_contribution_token_purchase_mint_for` |        | X               | X                     | X                      |
-//!
-//! _* They can call contribute only if the project is on the remainder round._
-//!
+//! 1) **Project Creation**: Issuer creates a project with the [`create_project`](Pallet::create_project) extrinsic.
+//! 2) **Evaluation Start**: Issuer starts the evaluation round with the [`start_evaluation`](Pallet::start_evaluation) extrinsic.
+//! 3) **Evaluate**: Evaluators bond PLMC to evaluate a project with the [`evaluate`](Pallet::evaluate) extrinsic.
+//! 4) **Evaluation End**: Anyone can end the evaluation round with the [`end_evaluation`](Pallet::end_evaluation) extrinsic after the defined end block.
+//! 5) **Auction Start**: If the project receives at least 10% of its target funding (in USD) in PLMC bonded, the auction starts immediately after `end_evaluation` is called
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:47]: here a dot is missing at the end of the sentence

File: pallets/funding/src/lib.rs [hunk 20]
```diff
@@ -18,96 +18,53 @@
...
 //!
 //! ## Overview
-//! The official logic for Polimec's blockchain can be found at our [whitepaper](https://polimec.link/whitepaper).
+//! The official logic for Polimec's blockchain can be found in our [knowledge hub](https://hub.polimec.org/).
 //!
 //! There are 3 types of users in Polimec:
 //! - **Issuers**: They create projects and are responsible for their success.
-//! - **Evaluators**: They evaluate projects and are rewarded for their work.
-//! - **Contributors**: They contribute financially to projects and are rewarded on the basis of their contribution
+//! - **Evaluators**: They are incentivized to assess projects accurately by locking their PLMC. If enough PLMC is locked,
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:31]: don't you want to specify "enough"? "at least 10% of its target funding (in USD) in PLMC bonded"

File: pallets/funding/src/lib.rs [hunk 21]
```diff
@@ -18,96 +18,53 @@
...
-//! | `vested_contribution_token_purchase_mint_for` |        | X               | X                     | X                      |
-//!
-//! _* They can call contribute only if the project is on the remainder round._
-//!
+//! 1) **Project Creation**: Issuer creates a project with the [`create_project`](Pallet::create_project) extrinsic.
+//! 2) **Evaluation Start**: Issuer starts the evaluation round with the [`start_evaluation`](Pallet::start_evaluation) extrinsic.
+//! 3) **Evaluate**: Evaluators bond PLMC to evaluate a project with the [`evaluate`](Pallet::evaluate) extrinsic.
+//! 4) **Evaluation End**: Anyone can end the evaluation round with the [`end_evaluation`](Pallet::end_evaluation) extrinsic after the defined end block.
+//! 5) **Auction Start**: If the project receives at least 10% of its target funding (in USD) in PLMC bonded, the auction starts immediately after `end_evaluation` is called
+//! 6) **Bid**: Professional and Institutional investors can place bids on the project using the [`bid`](Pallet::bid) extrinsic. the price starts at the issuer-defined minimum, and increases by increments of 10% in price and bucket size.
```
Comment (simonpolimec) [inline pallets/funding/src/lib.rs:48]: "institutional" investors (small letter); "The" price (capital letter) as it is the beginning of a sentence

**Review by lrazovic — APPROVED** [review]

**Comment by JuaniRios** [comment]
### Merge activity

* **Sep 10, 4:36 AM EDT**: @JuaniRios started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Polimec/polimec-node/386).

**Pull request merged**